edpm_ansible/ansible_plugins/modules/container_config_hash.py

```diff
@@ -209,29 +235,26 @@ def _update_hashes(self):
                 continue
             startup_config_json = json.loads(self._slurp(config))
             config_volumes = self._match_config_volumes(startup_config_json)
-            config_hashes = [
-                self._get_config_hash(vol_path) for vol_path in config_volumes
-            ]
-            config_hashes = filter(None, config_hashes)
-            if 'environment' in startup_config_json:
-                old_config_hash = startup_config_json['environment'].get(
-                    'EDPM_CONFIG_HASH', '')
-            if config_hashes is not None and config_hashes:
-                config_hash = '-'.join(config_hashes)
-                if config_hash == old_config_hash:
+            if config_volumes:
+                if 'environment' in startup_config_json:
```

Comment:
Current logic does not initialize old_config_hash and exclude_from_hash in case environment key does not exist, which causes reference without assignment.

Reply:
uh, thanks!